railties/lib/rails/generators/app_base.rb

```diff
@@ -324,7 +324,7 @@ def javascript_gemfile_entry
                                        "Use #{options[:javascript]} as the JavaScript library")
 
           unless options[:skip_turbolinks]
-            gems << GemfileEntry.version("turbolinks", "~> 5.x",
+            gems << GemfileEntry.version("turbolinks", "~> 5",
```

Comment:
Do we want to people upgrade to 6 without being careful? I prefer a more conservative version like `~> 5.0`